csm/conf/post_install.py

```diff
@@ -40,6 +40,11 @@ class PostInstall(Setup):
     def __init__(self):
         """Instiatiate Post Install Class."""
         Log.info("Executing Post Installation for CSM.")
+        self._replacement_node_flag = os.environ.get(
+            "REPLACEMENT_NODE") == "true"
+        if self._replacement_node_flag:
+            Log.info("REPLACEMENT_NODE flag is set")
```

Comment:
Remove node replacement related code for now... we still not designed the node replacement scenario.. We will take this up later